components/datasets/form/DatasetsForm.js

```diff
@@ -117,11 +117,16 @@ class DatasetsForm extends React.Component {
             omit: (dataset) ? ['connectorUrlHint', 'authorization', 'connectorType', 'provider'] : ['connectorUrlHint', 'authorization']
           };
 
+          const bodyObj = omit(this.state.form, requestOptions.omit);
+          const newSubscribable = {};
```

Comment:
```
          bodyObj.subscribable = bodyObj.subscribable.reduce((o, val) => ({
            ...o,
            [val.type]: val.value
          }), {});
```

Reply:
I think that this is beautiful and without eslint issues...

Reply:
👍

Reply:
👍